### PR TITLE
improve admin for aperture photometry results

### PR DESCRIPTION
**Pull request by juanep97**
Improves the admin for aperture photometry results, allows visually inspecting the apertures taken for each result.

### CODE REVIEW

File: iop4admin/modeladmins/aperphotresult.py [hunk 1]
```diff
@@ -97,4 +123,8 @@ def get_bkg_flux_counts_err(self, obj):
             return "-"
         else:
             return f"{obj.bkg_flux_counts_err:.1f}"
-    
\ No newline at end of file
+    
+    @admin.display(description="img")
+    def get_image_preview(self, obj, allow_tags=True):
+        url_img_preview = reverse(f"iop4admin:iop4api_{self.model._meta.model_name}_preview", args=[obj.id])
```
Comment (morcuended) [inline iop4admin/modeladmins/aperphotresult.py:129]: why the reverse?

File: iop4lib/db/aperphotresult.py [hunk 2]
```diff
@@ -6,9 +6,20 @@
 from django.db import models
 
 # other imports
+import os
+import os
```
Comment (morcuended) [inline iop4lib/db/aperphotresult.py:10]:
```suggestion
```

**Review by morcuended — DISMISSED** [review]
some minor stuff, looks good otherwise

**Review by morcuended — APPROVED** [review]

File: iop4admin/modeladmins/aperphotresult.py [hunk 3]
```diff
@@ -17,12 +17,26 @@
...
+    def get_urls(self):
+        urls = super().get_urls()
+        my_urls = [
+            path('preview/<int:pk>', self.admin_site.admin_view(self.view_preview),  name=f"iop4api_{self.model._meta.model_name}_preview"),
+        ]
+        return my_urls + urls
+    
+    def view_preview(self, request, pk):
+        
+        if ((fit := self.model.objects.filter(id=pk).first()) is None):
```
[the remark this answers is not in the document]
Comment (juanep97) [reply iop4admin/modeladmins/aperphotresult.py:35]: It was

**Pull request merged**